Apps/APN/ios/AppDelegate.mm

```diff
@@ -8,6 +8,7 @@
 #import <UserNotifications/UserNotifications.h>
 #import <RNCPushNotificationIOS.h>
 #import <React/RCTLinkingManager.h>
+#import "RNNotifications.h"
```

Comment:
Modifications in this file were from react-native-notifications docs: https://wix.github.io/react-native-notifications/docs/getting-started